packages/core/src/plugins/default/markbind-plugin-tree.css

```diff
@@ -0,0 +1,6 @@
+.tree {
+    white-space: pre;
+    font-family: "Fira Code", "Fira Mono", monospace;
+    font-size: 0.9375rem;
+    line-height: 1.5;
+}
```

Comment:
Looks like the `tree` can overflow the page similar to the embed issue:

![image](https://user-images.githubusercontent.com/47470981/157467884-29f09159-f66d-4bdb-b974-f516a2bf4147.png)

Maybe can try adding `overflow-x: auto`?

Relevant code:
```.html
<tree>
Edit me to generate
  a
    nice
      tree
        diagram!
        :)
  b
    c
      d
        e
          f
            g
              h
                a very long file name
</tree>
```